module-apps/apps-common/widgets/DigitsContainer.hpp

```diff
@@ -16,47 +16,20 @@ namespace gui
         std::uint32_t minusWidth;
         std::uint32_t leftBoxWidth;
         std::uint32_t rightBoxWidth;
+        std::uint32_t avaliableEvenWidth;
+        bool notCentral{false};
```

Comment:
what this flag does? Basing on the name, I can't figure it out at the first look. Consider renaming to `isNotCentered` or something like that